Registry/Registry.EM_COMMON

```diff
@@ -1189,7 +1189,12 @@ state   real    F_ICE_PHY      ikj     misc         1         -      rhdu     "F
 state   real    F_RAIN_PHY     ikj     misc         1         -      rhdu     "F_RAIN_PHY"           "FRACTION OF RAIN "         ""
 state   real    F_RIMEF_PHY    ikj     misc         1         -      rhdu     "F_RIMEF_PHY"          "MASS RATIO OF RIMED ICE "         ""
 state   real    qndropsource   ikj     misc         1         -      r        "qndropsource"         "Droplet number source"   "#/kg/s"
-
+# ivanr microphysics variables
+state    real    cld           ikj    misc        1         -      r        "CLD"                  "w2id "         ""
+state    real    tw2           ikj    misc        1         -      r        "TW2"                  "w2id "         ""
+state    real    pw2           ikj    misc        1         -      r        "PW2"                  "w2id "         ""
+state    real    qw2           ikj    misc        1         -      r        "QW2"                  "w2id "         ""
+state    real    f_snow        ikj    misc        1         -      r        "F_SNOW"               "w2id "         ""
```

Comment:
@Nikolar97 
Please include a description of these fields with units. An example is the line directly above.
Since these are "r" restart variables, have you tested that you do indeed get reproducible results with a restart?

Reply:
@davegill 
Added the descriptions to the fields in the EM_COMMON and NMM registry. The variables should be r, they were used in a similar way to other r fields in other custom schemes already in the model.